extern/drawdance/cmake/compile_flags.cmake

```diff
@@ -22,16 +22,28 @@ endif()
 
 if(MSVC)
     # C4100: unreferenced formal parameter
-    #  - MSVC has no `__attribute__((unused))` and parameter names are not
-    #    omitted from the code
-    # C4200: nonstandard extension used: zero-sized array in struct/union
-    #  - MSVC does not recognise C99 flexible array members as standard
+    #  - MSVC has no `__attribute__((unused))` and omitting parameter names
+    #    wouldn't be standard C
+    # C4123: const object should be initialized
+    #  -  warns about static values without an initializer, which is nonsense
+    #     because static values are zero-initialized
     # C4127: conditional expression is constant
-    #  - macros from cthash emit `do {} while(0);` and this triggers on
+    #  - macros from uthash emit `do {} while(0);` and this triggers on
     #    `while(0)`
+    # C4200: nonstandard extension used: zero-sized array in struct/union
+    #  - MSVC does not recognise C99 flexible array members as standard
+    # C4310: cast truncates constant value
+    #  - this warning is just straight-up lies, it warns about stuff like
+    #    (char)0x80, which doesn't truncate at all, it reinterprets the bits
```

Comment:
I'll see if it likes that better, since the other compilers don't care either way. The case I saw the warning was for platform-specific SIMD code, the x86 intrinsics headers stupidly declare their octet parameters as a bare `char` even though they're really unsigned, so you have to do these bit reinterpretations.